service/src/main.rs

```diff
@@ -751,34 +786,24 @@ fn ensure_account_has_funds(api: &mut Api<sr25519::Pair, WsRpcClient>, accountid
 	let nonce = api.get_nonce_of(&alice_acc).unwrap();
 	info!("    Alice's Account Nonce is {}", nonce);
 
-	// check account balance
-	let free = api.get_free_balance(accountid).unwrap();
-	info!("TEE's free balance = {:?}", free);
-
-	let existential_deposit = api.get_existential_deposit().unwrap();
-	info!("Existential deposit is = {:?}", existential_deposit);
-	let funding_amount = existential_deposit * MIN_FUND_INCREASE_FACTOR - free;
-	info!("Funding amount is = {:?}", funding_amount);
 	if funding_amount > alice_free.unwrap() {
 		error!(
 			"funding amount is to high: please change MIN_FUND_INCREASE_FACTOR ({:?})",
 			funding_amount
 		);
 	}
 
-	if free < funding_amount {
-		let signer_orig = api.signer.clone();
-		api.signer = Some(alice);
+	let signer_orig = api.signer.clone();
+	api.signer = Some(alice);
 
-		println!("[+] bootstrap funding Enclave form Alice's funds");
-		let xt = api.balance_transfer(GenericAddress::Id(accountid.clone()), funding_amount);
-		let xt_hash = api.send_extrinsic(xt.hex_encode(), XtStatus::InBlock).unwrap();
-		info!("[<] Extrinsic got finalized. Hash: {:?}\n", xt_hash);
+	println!("[+] bootstrap funding Enclave from Alice's funds");
+	let xt = api.balance_transfer(GenericAddress::Id(accountid.clone()), funding_amount);
+	let xt_hash = api.send_extrinsic(xt.hex_encode(), XtStatus::InBlock).unwrap();
+	info!("[<] Extrinsic got finalized. Hash: {:?}\n", xt_hash);
```

Comment:
```suggestion
	info!("[<] Extrinsic got included in a block. Hash: {:?}\n", xt_hash);
```

Reply:
This is an important distinction 👍 included in a block is not the same as finalized